lib/rubocop/cop/layout/ordered_methods.rb

```diff
@@ -81,13 +81,11 @@ def cache(node)
           end
         end
 
-        def consecutive_methods(nodes)
+        def consecutive_methods(nodes, &block)
           filtered = filter_relevant_nodes(nodes)
           filtered_and_grouped = group_methods_by_access_modifier(filtered)
           filtered_and_grouped.each do |method_group|
-            method_group.each_cons(2) do |left_method, right_method|
-              yield left_method, right_method
-            end
+            method_group.each_cons(2, &block)
```

Comment:
I'd like to keep the `yield` here for performance reasons. I've seen benchmarks that show using `&block` to be 5x slower than using `yield` without a `&block`. Since this is called on using each method in a linting scan, I'd like it to be optimized for performance.

Here's a post from 2016 with a benchmarking script with results that are close to what I get on Ruby 2.7 and 3

Reply:
> I'd like to keep the `yield` here for performance reasons. 

👍 Resolved by 71178b117f8708fe